docs/src/pages/configurations/typescript-config/index.md

```diff
@@ -0,0 +1,229 @@
+---
+id: 'typescript-config'
+title: 'Typescript Config'
+---
+
+This is a central reference for using Storybook with Typescript.
+
+## Dependencies you may need
+
+```bash
+yarn add -D typescript
+yarn add -D awesome-typescript-loader # alternative to ts-loader
```

Comment:
are these two loaders interchangeable in this guide? we should leave a note about it.

Reply:
yes